src/routes/game/GameView.vue

```diff
@@ -312,7 +312,7 @@
       </div>
 
       <h3 id="player-score">
-        <span>POINTS: {{ gameStore.playerPointTotal }}</span>
+        <span>{{ t('game.counter.points') }}: {{ gameStore.playerPointTotal }}</span>
```

Comment:
This is a nitpick, but because `counter` has a different specific meaning in the game (you can counter someone's one-off effect by playing a 2), let's change the key name here to 'game.score' instead of 'game.counter'
```suggestion
        <span>{{ t('game.score.points') }}: {{ gameStore.playerPointTotal }}</span>
```